Marlin/stepper.cpp

```diff
@@ -1256,10 +866,14 @@ void quickStop() {
 
 #endif //BABYSTEPPING
 
+/**
+ * Software-controlled Stepper Motor Current
+ */
+
 #if HAS_DIGIPOTSS
 
   // From Arduino DigitalPotControl example
-  void digitalPotWrite(int address, int value) {
+  Stepper::void digitalPotWrite(int address, int value) {
```

Comment:
Is void Stepper::digitalPotWrite(int address, int value) { right? ([same comment in PR #3631](https://github.com/MarlinFirmware/Marlin/pull/3631#discussion_r61534605))